docs/configuration/config-file-reference.md

```diff
@@ -104,6 +104,20 @@ Where default_value is the value to use if the environment variable is undefined
 # The table_manager_config configures the Cortex table-manager.
 [table_manager: <table_manager_config>]
 
+data_purger:
```

Comment:
Do you mind adding a new root block to the doc generator, so that the `data_purger_config` will have its own dedicated section in this reference file? It's 3 lines config at the top of `tools/doc-generator/main.go` (look at other root blocks as a reference)